internal/generator/services.go

```diff
@@ -35,6 +35,16 @@ var supportedAutogeneratedTypes = []string{
 	"python",
 }
 
+// these are lagoon types that support autogenerated routes
+var supportedDBTypes = []string{
+	"mariadb",
+	"mariadb-dbaas",
+	"postgres",
+	"postgres-dbaas",
+	"mongodb",
+	"mongodb-dbaas",
```

Comment:
There may also be a "mongo" type as per https://github.com/uselagoon/build-deploy-tool/blob/main/legacy/build-deploy-docker-compose.sh#L558

and this is how we built the internal-services-test (https://github.com/uselagoon/internal-services-test/blob/main/docker-compose.yml#L81)

Reply:
:hidethepain:

Reply:
(confirmed that `lagoon.type: mongo` is the more commonly used option, but it should also work for mongodb too